onyo/lib/onyo.py

```diff
@@ -491,7 +491,7 @@ def get_template(self,
         return yaml_to_dict(template_file)
 
     def validate_anchors(self) -> bool:
-        """Check if all dirs (except those in `.onyo/`) contain an .anchor file.
+        r"""Check if all dirs (except those in `.onyo/`) contain an .anchor file.
```

Comment:
`.anchor` in backticks?

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.